src/settings/RoutingListConfiguration/RoutingListConfigurationView.js

```diff
@@ -0,0 +1,109 @@
+import get from 'lodash/get';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+  injectIntl,
+} from 'react-intl';
+import {
+  useHistory,
+  useLocation,
+} from 'react-router';
+
+import {
+  Accordion,
+  AccordionSet,
+  AccordionStatus,
+  Button,
+  Col,
+  ExpandAllButton,
+  KeyValue,
+  Pane,
+  Row,
+} from '@folio/stripes/components';
+import { ViewMetaData } from '@folio/stripes/smart-components';
+
+import RoutingListConfigurationEmailTemplate from './RoutingListConfigurationEmailTemplate';
+
+const RoutingListConfigurationView = (props) => {
+  const location = useLocation();
+  const history = useHistory();
+  const {
+    listConfig,
+    intl: {
+      formatMessage,
+    },
+  } = props;
+
+  const emailTemplate = get(listConfig, 'localizedTemplates.en.body', '');
+
+  const onEdit = () => {
+    const editPath = `${location.pathname}/edit`;
+
+    history.push(editPath);
+  };
+
+  const renderEditIcon = () => {
```

Comment:
If the pane doesn't accept a function as a prop, wouldn't it be simpler to keep JSX in the constant instead of a function in this scenario?